src/xtgeo/surface/regular_surface.py

```diff
@@ -1086,6 +1086,7 @@ def to_file(
         fformat: Optional[str] = "irap_binary",
         pmd_dataunits: Optional[Tuple[int, int]] = (15, 10),
         engine: Optional[str] = "cxtgeo",
+        error_if_near_empty: bool = False,
```

Comment:
Will consider; thanks

Reply:
After considering: keep as is for now, since it will require an additional keyword. Want to keep the bool option to either warn or raise error, for backward compatibility a warning is default (i.e. not breaking current implementation for clients)